provider/lxd.go

```diff
@@ -483,3 +485,9 @@ func (l *LXD) Stop(ctx context.Context, instance string, force bool) error {
 func (l *LXD) Start(ctx context.Context, instance string) error {
 	return l.setState(ctx, instance, "start", false)
 }
+
+// GetVersion returns the interface version of the provider.
+func (l *LXD) GetVersion(ctx context.Context) string {
+	fmt.Println(version)
```

Comment:
This doesn't feel right. We either return a string or just print it, but not both. This function gets called in garm-provider-common here:

https://github.com/cloudbase/garm-provider-common/blob/899c120c80ce01a42e94cc2319e1e1df6407e4ff/execution/v0.1.0/execution.go#L159-L161

Which in turn gets used by the provider here:

https://github.com/cloudbase/garm-provider-lxd/blob/b63472c5becad2b9a39603e37f67f26d361e0d30/main.go#L55

the result of which is printed here:

https://github.com/cloudbase/garm-provider-lxd/blob/b63472c5becad2b9a39603e37f67f26d361e0d30/main.go#L60-L62

if you test these changes with:

```bash
GARM_COMMAND="GetVersion" ./garm-provider-lxd
```

you will most likely get the version printed twice.